java/ingest/ingest-batcher-core/src/main/java/sleeper/ingest/batcher/IngestBatcher.java

```diff
@@ -76,27 +76,59 @@ public static Builder builder() {
 
     public void batchFiles() {
         Instant time = timeSupplier.get();
-        store.getPendingFilesOldestFirst().stream()
-                .collect(Collectors.groupingBy(FileIngestRequest::getTableName, LinkedHashMap::new, toList()))
-                .forEach((tableName, inputFiles) -> batchTableFiles(tableName, inputFiles, time));
+        List<FileIngestRequest> pendingFiles = store.getPendingFilesOldestFirst();
+        if (pendingFiles.isEmpty()) {
+            LOGGER.info("No pending files found");
+        } else {
```

Comment:
Can we log the number of pending file ingest requests in both branches of the if?